src/app/api/comapnies/services/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Company, PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export async function getAllCompanies() {
```

Comment:
It would be better if we add 'Service' to the function name, e.g., getAllCompaniesService, to distinguish what the function really does.